products/BellHybrid/apps/application-bell-focus-timer/windows/FocusSettingsWindow.cpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2017-2024, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "FocusSettingsWindow.hpp"
+
+#include <common/data/StyleCommon.hpp>
+#include <apps-common/ApplicationCommon.hpp>
+#include <module-gui/gui/input/InputEvent.hpp>
+#include <module-gui/gui/widgets/SideListView.hpp>
+
+namespace app::focus
+{
+    using namespace gui;
+    SettingsWindow::SettingsWindow(app::ApplicationCommon *app,
+                                   std::unique_ptr<contract::SettingsPresenter> presenter,
+                                   const std::string name)
+        : AppWindow(app, std::move(name)), presenter{std::move(presenter)}
```

Comment:
`name` won't actually be moved here, I'd change the argument type to `const &` and remove this `move` or remove the `const` completely

Reply:
done